src/plugins/vuetify.js

```diff
@@ -1,8 +1,9 @@
+import '@mdi/font/css/materialdesignicons.css';
 import Vue from 'vue';
 import Vuetify from 'vuetify/lib/framework';
 
 Vue.use(Vuetify);
 
 export default new Vuetify({
-    theme: { dark: true },
+    theme: { dark: true }
```

Comment:
Trailing commas should only be removed when you want to signal an intention never to add a line after this one, which isn't the case here.  (This is a personal opinion, but a strong one I've never heard a decent counter-argument to)

Imagine you wanted to add another config option after `theme`.  You would need to add a trailing comma, which creates an unnecessary extra line to review.

Also, not relevant here, but if you ever need to auto-generate code, it's much easier to print a list of values if you don't have to worry about that special case.